packages/trader/src/sass/app/_common/components/contract-type-info.scss

```diff
@@ -51,7 +51,12 @@
             @include typeface(--paragraph-left-normal-black, none);
             margin-bottom: 1.6rem;
             color: var(--text-general);
+
+            @supports (-moz-appearance: none) {
```

Comment:
@henry-deriv i can see that you are changing the width of certain content elements but in the future the content may be changed, will it work to apply this to &__content itself?

Reply:
sounds good